.github/workflows/standard-deploy.yml

```diff
@@ -71,12 +62,10 @@ jobs:
 
     - name: Inject replacement image into manifest
       run: |
-        export IMAGE_LOCATION="${{ inputs.image_location || format('ghcr.io/communitiesuk/funding-service-design-{0}:{1}', inputs.app_name, inputs.version) }}"
-        yq -i ".image.location = \"${IMAGE_LOCATION}\""  copilot/fsd-${{ inputs.app_name }}/manifest.yml
+        yq -i ".image.location = \"${{ inputs.image_location }}\""  copilot/fsd-${{ inputs.app_name }}/manifest.yml
 
     - name: Run database migrations
-      if: ${{ inputs.db_name != '' }}
-      run: scripts/migration-task-script.py ${{ inputs.environment }} ${{ inputs.db_name }}
+      run: scripts/migration-task-script.py ${{ inputs.environment }} ${{ inputs.image_location }}
```

Comment:
We used to have that `inputs.db_name` check on running this step of the job so only stores/back-end services that passed that variable ran this step - now we've dropped that, what happens when one of the front-end apps calls this shared workflow and doesn't have the `migration-task-script` present?

Reply:
👀  Good spot.

We still need some kind of flag for DB-apps. Let me add something...